semantic-tsx/src/Language/TSX/Tags.hs

```diff
@@ -222,7 +235,7 @@ instance ToTags Tsx.MemberExpression
 instance ToTags Tsx.MetaProperty
 -- instance ToTags Tsx.MethodDefinition
 instance ToTags Tsx.MethodSignature
-instance ToTags Tsx.Module
+-- instance ToTags Tsx.Module
```

Comment:
Cut this comment?

Reply:
I had been leaving them in intentionally to signal their place in the alphabetized list but I’m not really fussed. But if we remove these we should remove _all_ of them, IMO.